src/Core/Grid/Query/CategoryQueryBuilder.php

```diff
@@ -208,4 +212,15 @@ private function getQueryBuilder(array $filters)
 
         return $qb;
     }
+
+    /**
+     * @param array $filters
+     *
+     * @return bool
+     */
+    private function isSearchRequestOnHomeCategory(array $filters)
+    {
+        return (isset($filters['is_home_category']) && ($filters['is_home_category'] === true))
+            && (isset($filters['is_search_request']) && ($filters['is_search_request'] === true));
```

Comment:
```suggestion
        return isset($filters['is_home_category'], $filters['is_search_request'])
             && $filters['is_home_category'] === true && $filters['is_search_request'] === true;
```

Reply:
Please tell me if I'm wrong, but I think this last feedback is not an issue in the PR but rather a personal preference 😉 and hence we should not block PRs for that. The goal of review is to validate the quality of a PR, not that it is written exactly as you would have written it yourself 😉

Reply:
It's rather an optimization : check if all exists before tests. In addition, that removes some code, and less code, less bugs.

Reply:
You are killing me 🙄

Reply:
Done